yt_dlp/extractor/zaiko.py

```diff
@@ -0,0 +1,55 @@
+import json
+import html
+import re
+
+from .common import InfoExtractor
+from ..utils import traverse_obj
+
+
+class ZaikoIE(InfoExtractor):
+    _VALID_URL = r'https://zaiko\.io/event/(?P<id>\d+)/stream(?:/\d+)+'
+    _TESTS = [{
+        'url': 'https://zaiko.io/event/324868/stream/20571/20571',
+        'info_dict': {
+            'id': 324868,
+            'ext': 'mp4',
+            'title': 'ZAIKO STREAMING TEST',
+            'uploader_id': 454,
+            'timestamp': 1583809200,
```

Comment:
make sure to actually run this test, right now it fails without an `upload_date` value

use `test/local_parameters.json` to pass cookies params for the test download, e.g.:
```json
{
  "cookiesfrombrowser": [
    "firefox",
    null,
    null,
    null
  ]
}
```